### PR TITLE
show thumbnail of scene on scene save as

### PR DESCRIPTION
**Pull request by aditya-mitra**
## Summary

Show the thumbnail of scene when the scene is saved as. Also, make the _saveasscene dialog_'s styling better.

#### Earliier

image

#### Now

image

### CODE REVIEW

File: packages/editor/src/components/EditorContainer.tsx [hunk 1]
```diff
@@ -184,11 +184,12 @@ const onSaveAs = async () => {
   try {
     if (sceneName || editorState.sceneModified.value) {
       const blob = await takeScreenshot(512, 320, 'ktx2')
+      const thumbnailBlob = await takeScreenshot(256, 160, 'jpeg')
```
Comment (HexaField) [inline packages/editor/src/components/EditorContainer.tsx:187]: we dont want to be taking two screenshots, we are able to display ktx2s as images

**Review by HexaField — CHANGES_REQUESTED** [review]
We should also include the thumbnail in the regular save menu.

**Review by mrhegemon — APPROVED** [review]

**Review by mrhegemon — APPROVED** [review]

File: packages/editor/src/functions/takeScreenshot.ts [hunk 2]
```diff
@@ -162,10 +162,10 @@ export async function previewScreenshot(
 export async function takeScreenshot(
   width: number,
   height: number,
-  format = 'ktx2' as 'png' | 'ktx2' | 'jpeg',
+  format = 'jpeg' as 'jpeg' | 'png',
```
Comment (mrhegemon) [inline packages/editor/src/functions/takeScreenshot.ts:165]: Huh? Why these formats?

**Review by mrhegemon — CHANGES_REQUESTED** [review]

**Comment by aditya-mitra** [comment]
> We should also include the thumbnail in the regular save menu.

Implemented!

image

**Review by HexaField — APPROVED** [review]

**Pull request merged**